static/js/click_handlers.js

```diff
@@ -468,7 +468,6 @@ $(function () {
             // wrong.
             if (this.hasChildNodes()) {
                 this.innerText = this.innerText;
-                place_caret_at_end(this);
```

Comment:
@brockwhittaker can you review this line?  I want to make sure this is the right fix.